dev-scripts/worktree-init.sh

```
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+cd -- "$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")/.." && pwd)"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)